RecoEgamma/EgammaElectronProducers/src/LowPtGsfElectronSeedHeavyObjectCache.cc

```diff
@@ -6,9 +7,8 @@
 #include "DataFormats/TrackReco/interface/TrackFwd.h"
 #include "FWCore/ParameterSet/interface/FileInPath.h"
 #include "RecoEgamma/EgammaElectronProducers/interface/LowPtGsfElectronSeedHeavyObjectCache.h"
-#include "RecoEgamma/EgammaTools/interface/GBRForestTools.h"
-#include "TMVA/MethodBDT.h"
-#include "TMVA/Reader.h"
+//#include "TMVA/MethodBDT.h"
+//#include "TMVA/Reader.h"
```

Comment:
the commented out lines better be removed.
I see that they are also in the master. A matching cleanup would help.